Description: Source code is using this #define for namespace mangling
 Thus it is important for ABI to remove the patch level
Author: Ollie Harding <o...@dneg.com>
Bug-Debian: http://bugs.debian.org/695665
Forwarded: https://groups.google.com/group/openvdb-forum/msg/475a50dd0fe19f4d

Index: openvdb/version.h
===================================================================
--- openvdb.orig/version.h	2013-12-13 01:35:59.000000000 +0100
+++ openvdb/version.h	2013-12-29 12:31:43.578961269 +0100
@@ -42,7 +42,7 @@
 /// vdb::vX_Y_Z::Vec3i, vdb::vX_Y_Z::io::File, vdb::vX_Y_Z::tree::Tree, etc.,
 /// where X, Y and Z are OPENVDB_LIBRARY_MAJOR_VERSION, OPENVDB_LIBRARY_MINOR_VERSION
 /// and OPENVDB_LIBRARY_PATCH_VERSION, respectively (defined below).
-#define OPENVDB_VERSION_NAME v2_1_0
+#define OPENVDB_VERSION_NAME v2_1
 
 // Library major, minor and patch version numbers
 #define OPENVDB_LIBRARY_MAJOR_VERSION_NUMBER 2
